app/cmd.go

```diff
@@ -524,6 +535,12 @@ func getConfig() (appConfig, error) {
 
 func init() {
 	RootCmd.Flags().StringP("listen", "l", "localhost:8081", "Listen address:port for the application.")
+	RootCmd.Flags().StringP("listen-tls", "t", "", "HTTPS listen address:port for the application.  Requires setting --tls-cert and --tls-key OR --tls-cert-file and --tls-key-file.")
+
+	RootCmd.Flags().String("tls-cert-file", "", "If using listen-tls, specify path of PEM-encoded certificate file.")
+	RootCmd.Flags().String("tls-key-file", "", "If using listen-tls, specify path of PEM-encoded private key file.")
+	RootCmd.Flags().String("tls-cert", "", "If using listen-tls, specify PEM-encoded certificate.")
```

Comment:
Same as above, also, should we call this `tls-cert-data` to be a little clearer?